htdocs/ticket/card.php

```diff
@@ -286,6 +286,11 @@
 			if (!$error) {
 				$db->commit();
 
+				// if a user got defined as an assigned user: run trigger
+				if (!empty($object->fk_user_assign)) {
```

Comment:
We already have a trigger for the create and it is job of the trigger to intercept also this case to scan if an assignement is done directly during creation and to execute its custom code.
Rule is to have only 1 trigger per business CRUD event, even when several actions are done in same transaction.

Reply:
Having only one trigger call makes perfect sense, thanks for the explanation. I added this rule also to the wiki.